app/javascript/.eslintrc.js

```diff
@@ -2,9 +2,14 @@ const path = require('path');
 
 module.exports = {
   parser: 'babel-eslint',
-  extends: ['airbnb', 'plugin:jsx-a11y/recommended', 'prettier'],
+  extends: [
+    'eslint:recommended',
```

Comment:
Using the eslint recommended config now along with the Preact eslint recommended configuration.